context_test.go

```diff
@@ -150,6 +150,19 @@ func TestContext_Value(t *testing.T) {
 	expect(t, c.Value("unknown-flag"), nil)
 }
 
+func TestContext_Value_InvalidFlagAccessHandler(t *testing.T) {
```

Comment:
Can you add a test with one app, one command and one subcommand.  ? I dont think it will work with the way you expect as of now since you have to set InvalidFlagAccessHandler in Command.startApp.

Reply:
Fixed it.